config/accounts.go

```diff
@@ -45,13 +45,16 @@ type CookieSync struct {
 type AccountCCPA struct {
 	Enabled            *bool              `mapstructure:"enabled" json:"enabled,omitempty"`
 	IntegrationEnabled AccountIntegration `mapstructure:"integration_enabled" json:"integration_enabled"`
```

Comment:
It might be nice to provide some kind of deprecation warning if `integration_enabled` is found in a fetched account config but that could really bog down the logs. We should give some thought to whether we should bother to do something like that, and if so, how.

Reply:
Discussed offline. We can't currently do this without taking a major performance hit. Throwing warnings with respect to JSON configs is something that can be introduced when we move from caching JSON to caching structs.

Reply:
You can add a metric to give the host an idea of legacy account parameters are being used.

Reply:
@AlexBVolcy I like @SyntaxNode's idea of adding a metric. I suggest doing this in a separate PR if possible.